core/templates/pages/exploration-editor-page/editor-navigation/editor-navigation.component.html

```
@@ -1,4 +1,4 @@
-<i class="fa fa-book show-mobile-navbar-icon e2e-test-mobile-options" (click)="toggleMobileNavOptions()" [ngClass]="{'mobile-navbar-toggled': mobileNavOptionsAreShown}" aria-label="Mobile navbar options" role="button"></i>
+<i class="fa fa-book show-mobile-navbar-icon e2e-test-mobile-options" (click)="toggleMobileNavOptions()" [class.mobile-navbar-toggled]="mobileNavOptionsAreShown" aria-label="Mobile navbar options" role="button"></i>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)